test/manual/kerberos.test.js

```diff
@@ -3,12 +3,31 @@ const { MongoClient } = require('../../src');
 const chai = require('chai');
 const expect = chai.expect;
 
+function verifyKerberosAuthentication(client, done) {
+  client
+    .db('kerberos')
+    .collection('test')
+    .find()
+    .toArray(function (err, docs) {
+      expect(err).to.not.exist;
+      expect(docs).to.have.length(1);
+      expect(docs[0].kerberos).to.be.true;
+      client.close(done);
```

Comment:
can this be updated to ensure that client.close executes even if the assertions fail? the easiest way would be to do something like: `let expectError; try { expect ... } catch (err) { expectError = err } client.close(err => { done(expectError || err) })`

Reply:
Just updated that so it closes it properly 👍

Reply:
Thank you very much!